<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InspectorAmbitus</class>
 <widget class="QWidget" name="InspectorAmbitus">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>266</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="accessibleName">
   <string>Ambitus Inspector</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="elementName">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="accessibleName">
      <string/>
     </property>
     <property name="text">
      <string>Ambitus</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>3</number>
     </property>
     <property name="rightMargin">
      <number>3</number>
     </property>
     <property name="horizontalSpacing">
      <number>3</number>
     </property>
     <property name="verticalSpacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_6">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Head group</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="noteHeadGroup">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="accessibleName">
        <string>Head group</string>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>Normal</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cross</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Diamond</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Triangle</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Slash</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>XCircle</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Do</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Re</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Mi</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Fa</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Sol</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>La</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ti</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Alt. Brevis</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDoubleSpinBox" name="lineWidth">
       <property name="accessibleName">
        <string>Line thickness</string>
       </property>
       <property name="suffix">
        <string>sp</string>
       </property>
       <property name="maximum">
        <double>10.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QToolButton" name="resetLineWidth">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Reset to default</string>
       </property>
       <property name="accessibleName">
        <string>Reset 'Line thickness' value</string>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/data/icons/edit-reset.svg</normalon>
        </iconset>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_9">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Bottom note</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QComboBox" name="bottomTpc">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="accessibleName">
          <string>Bottom note</string>
         </property>
         <property name="currentText">
          <string notr="true">---</string>
         </property>
         <property name="maxVisibleItems">
          <number>15</number>
         </property>
         <property name="maxCount">
          <number>40</number>
         </property>
         <item>
          <property name="text">
           <string notr="true">---</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>C♭♭</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>C♭</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>C</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>C♯</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>C♯♯</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>D♭♭</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>D♭</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>D</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>D♯</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>D♯♯</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>E♭♭</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>E♭</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>E</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>E♯</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>E♯♯</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>F♭♭</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>F♭</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>F</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>F♯</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>F♯♯</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>G♭♭</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>G♭</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>G</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>G♯</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>G♯♯</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>A♭♭</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>A♭</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>A</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>A♯</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>A♯♯</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>B♭♭</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>B♭</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>B</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>B♯</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>B♯♯</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Oct.</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="bottomOctave">
         <property name="accessibleName">
          <string>Oct</string>
         </property>
         <property name="minimum">
          <number>-1</number>
         </property>
         <property name="maximum">
          <number>8</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QCheckBox" name="hasLine">
       <property name="focusPolicy">
        <enum>Qt::TabFocus</enum>
       </property>
       <property name="accessibleName">
        <string>Has line</string>
       </property>
       <property name="text">
        <string>Has line</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QToolButton" name="resetHasLine">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Reset to default</string>
       </property>
       <property name="accessibleName">
        <string>Reset 'Has line' value</string>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="icon">
        <iconset resource="../musescore.qrc">
         <normaloff>:/data/icons/edit-reset.svg</normaloff>:/data/icons/edit-reset.svg</iconset>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Line thickness</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_8">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Top note</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QComboBox" name="topTpc">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="accessibleName">
          <string>Top note</string>
         </property>
         <property name="editable">
          <bool>false</bool>
         </property>
         <property name="currentText">
          <string notr="true">---</string>
         </property>
         <property name="maxVisibleItems">
          <number>15</number>
         </property>
         <property name="maxCount">
          <number>40</number>
         </property>
         <item>
          <property name="text">
           <string notr="true">---</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>C♭♭</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>C♭</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>C</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>C♯</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>C♯♯</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>D♭♭</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>D♭</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>D</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>D♯</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>D♯♯</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>E♭♭</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>E♭</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>E</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>E♯</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>E♯♯</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>F♭♭</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>F♭</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>F</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>F♯</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>F♯♯</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>G♭♭</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>G♭</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>G</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>G♯</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>G♯♯</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>A♭♭</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>A♭</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>A</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>A♯</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>A♯♯</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>B♭♭</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>B♭</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>B</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>B♯</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>B♯♯</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Oct.</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="topOctave">
         <property name="accessibleName">
          <string>Oct</string>
         </property>
         <property name="minimum">
          <number>-1</number>
         </property>
         <property name="maximum">
          <number>8</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="noteHeadType">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="accessibleName">
        <string>Head type</string>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>Auto</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Whole</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Half</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Quarter</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Breve</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QToolButton" name="resetNoteHeadGroup">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Reset to default</string>
       </property>
       <property name="accessibleName">
        <string>Reset 'Head group' value</string>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="icon">
        <iconset resource="../musescore.qrc">
         <normaloff>:/data/icons/edit-reset.svg</normaloff>:/data/icons/edit-reset.svg</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_7">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Head type</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QToolButton" name="resetDirection">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Reset to default</string>
       </property>
       <property name="accessibleName">
        <string>Reset 'Direction' value</string>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="icon">
        <iconset resource="../musescore.qrc">
         <normaloff>:/data/icons/edit-reset.svg</normaloff>:/data/icons/edit-reset.svg</iconset>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="3">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="updateRange">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="focusPolicy">
          <enum>Qt::TabFocus</enum>
         </property>
         <property name="accessibleName">
          <string>Update range</string>
         </property>
         <property name="text">
          <string>Update Range</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="1" column="2">
      <widget class="QToolButton" name="resetNoteHeadType">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Reset to default</string>
       </property>
       <property name="accessibleName">
        <string>Reset 'Head type' value</string>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="icon">
        <iconset resource="../musescore.qrc">
         <normaloff>:/data/icons/edit-reset.svg</normaloff>:/data/icons/edit-reset.svg</iconset>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Direction</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="direction">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="accessibleName">
        <string>Direction</string>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>Upright</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Leaning Left</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Leaning Right</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>noteHeadGroup</tabstop>
  <tabstop>resetNoteHeadGroup</tabstop>
  <tabstop>noteHeadType</tabstop>
  <tabstop>resetNoteHeadType</tabstop>
  <tabstop>direction</tabstop>
  <tabstop>resetDirection</tabstop>
  <tabstop>hasLine</tabstop>
  <tabstop>resetHasLine</tabstop>
  <tabstop>lineWidth</tabstop>
  <tabstop>resetLineWidth</tabstop>
  <tabstop>topTpc</tabstop>
  <tabstop>topOctave</tabstop>
  <tabstop>bottomTpc</tabstop>
  <tabstop>bottomOctave</tabstop>
 </tabstops>
 <resources>
  <include location="../musescore.qrc"/>
 </resources>
 <connections/>
</ui>
